### ISSUE TITLE
Infinite loop without scroll to first slide

### ISSUE THREAD

**Issue by martin-badin** [SSENSE/vue-carousel#288]
Please add support for infinite loop without scroll to the first slide.

If I'm on the last slide, the next slide should be the first slide. For example please see [the slick slider](http://kenwheeler.github.io/slick/).

**Comment by focussing** [comment]
👍

**Comment by sergiocastrovale** [comment]
Yeah, this is kind of the expected behavior on a loop-able slider, not the other way around. Any chance it will be included soon?

**Comment by quinnlangille** [comment]
We've had lots of requests for this, I'll flag this as a feature request!

**Comment by wes977** [comment]
I added a story to the play using a prop "loop" that makes the carousel return to the first slide if set to true.

**Comment by focussing** [comment]
yes that would be much appreciated :)

**Comment by Kemper-Diao** [comment]
This request is solved ?  I have the same request

**Comment by bartdominiak** [comment]
@Kemper-Diao Use prop: `loop` and set it to `true`

**Comment by Kemper-Diao** [comment]
> 
> 
> @Kemper-Diao Use prop: `loop="true"`

but It's still not working, my version is  0.16.0-rc1, is that right?

**Comment by quinnlangille** [comment]
Hey @Kemper-Diao, no this is still an open feature request. If a feature has been added into a release, it's issue will be closed and comment will be made with the release.

**Comment by dacsang97** [comment]
@Kemper-Diao `:loop="true"` working in 0.16.0-rc2

**Comment by simonorono** [comment]
@timeandtimeago this is yet to be implemented

**Comment by engelde** [comment]
Just wanted to throw in another request for this feature. This is a great project. Thank you and keep up the good work!

**Comment by Nfinished** [comment]
+1

**Comment by gasperecf** [comment]
This would also solve problem with swipe to next on last slide

**Comment by Lanatel** [comment]
+1

**Comment by damienmcd** [comment]
+1

**Comment by jfaq89** [comment]
+1

**Comment by hcristiano** [comment]
+1

**Comment by madhuab** [comment]
Is this feature released? _1-2-3-1-2-3-1-2-3 not 1-2-3-321-2-3-321-2-3_

**Comment by cnsaavedra** [comment]
+1 for (say for example 3 slides) loopable infinitely instead of scrolling from 3 to 1 by going through 2

i.e.:
. . . 3-> 1 -> 2 -> 3 -> 1 -> 2 -> 3 . . .

**Comment by Skmt3P** [comment]
When will this PR be merged...? Is there anything that I can do?
https://github.com/SSENSE/vue-carousel/pull/360

**Comment by abhipanda** [comment]
Its over a year now :(

**Comment by marcosdipaolo** [comment]
Any news about this?

**Comment by Oswalman** [comment]
I have the same problem :c

**Comment by jeff-kroot** [comment]
Everyone here, see my comment in #360. There are bugs in that PR when combined with `:scrollPerPage="false"`. If you don't care about that, my suggestion would be to fork, apply the patch, and deploy it for yourself.

**Comment by yohanesisa** [comment]
i just use vue-carousel today and wondering if it can do a infinite loop, the documentation mention nothing about it and i found this open ticket...

so i'm using `"vue-carousel": "^0.18.0"`
and i used props `<carousel :loop="true">` and it doing great!

Hope everyone that wondering can try it too and documentation should be update soon.
Thank you for this great package, so appreciate it 👌

**Comment by focussing** [comment]
You are right; it works.
But what it does it goes to the last slide and then runs back in the opposite direction to slide 1.
Instead of keeping the scrolling direction as it was…

Best regards, 
Mit freundlichen Grüßen,
Met vriendelijke groet,
Raymond Verbruggen
 
FOCUSSING B.V.
Mispel 8
7621 WB Borne
Netherlands
+31 6 250 500 69 
mail@focussing.nl 
www.focussing.nl

**Comment by ogldev** [comment]
> Everyone here, see my comment in #360. There are bugs in that PR when combined with `:scrollPerPage="false"`. If you don't care about that, my suggestion would be to fork, apply the patch, and deploy it for yourself.

I forked the PR but how do you actually make it work? I couldn't make it infinite scrollable.
